.github/CODEOWNERS

```
@@ -7,6 +7,7 @@
 
 # Community partners
 /algorithmia/                            @koverholt support@algorithmia.io
+/superwise/                            @oryan-omer support@superwise.ai
```

```suggestion
/algorithmia/                            @koverholt support@algorithmia.io
/superwise/                              @oryan-omer support@superwise.ai
```
nit- let's keep the codeowners formatting consistent